docs/UserGuide.md

```diff
@@ -47,102 +47,102 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
 
-### 2.2. Adding a person: `add`
+### 2.2. Adding a task: `add`
 
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...`
+Adds a task to the address book<br>
```

Comment:
how about different variants? with deadlines, with start and end date/times etc?
need to capture all that the user can do